src/supertux/menu/cheat_apply_menu.cpp

```diff
@@ -0,0 +1,89 @@
+//  SuperTux
+//  Copyright (C) 2021 A. Semphris <semphris@protonmail.com>
+//
+//  This program is free software: you can redistribute it and/or modify
+//  it under the terms of the GNU General Public License as published by
+//  the Free Software Foundation, either version 3 of the License, or
+//  (at your option) any later version.
+//
+//  This program is distributed in the hope that it will be useful,
+//  but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+//  GNU General Public License for more details.
+//
+//  You should have received a copy of the GNU General Public License
+//  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+#include "supertux/menu/cheat_apply_menu.hpp"
+
+#include "gui/menu_item.hpp"
+#include "gui/menu_manager.hpp"
+#include "object/player.hpp"
+#include "scripting/functions.hpp"
+#include "supertux/game_session.hpp"
+#include "supertux/sector.hpp"
+
+CheatApplyMenu::CheatApplyMenu(std::function<void(Player&)> callback) :
+  m_callback_1(callback),
+  m_callback_2(nullptr),
+  m_stack_count(-1)
+{
+  add_label(_("Apply cheat to player"));
+  add_hl();
+
+  add_entry(-1, _("All Players"));
+  for (const auto player : Sector::get().get_players())
+  {
+    add_entry(player->get_id(), _("Player") + " " +
```

Comment:
Can you use some kind of string::format call? One that our translation functions can understand?

Reply:
Yea, it was the second thing I had to fix but I was too lazy to implement it now :smile: